x-pack/plugins/monitoring/server/lib/kibana/get_kibanas_for_clusters.js

```diff
@@ -5,7 +5,9 @@
  */
 
 import Bluebird from 'bluebird';
-import { chain, find, get } from 'lodash';
+import chain from 'lodash/chain';
```

Comment:
I'm seeing this error with this change:
```
server    log   [13:32:08.847] [error][monitoring][monitoring][plugins] TypeError: (0 , _chain.default)(...).map is not a function
    at callWithRequest.then.result (/Users/chris/dev/repos/kibana/x-pack/plugins/monitoring/server/lib/kibana/get_kibanas_for_clusters.js:183:12)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

Reply:
Is this related to point no 1 in @dgieselaar comment?: https://github.com/elastic/kibana/pull/74539#issuecomment-672808058

Reply:
Yes, thank you! Same thing indeed